src/CADL.Extension/Emitter.Csharp/src/type/inputType.ts

```diff
@@ -49,6 +49,7 @@ export interface InputModelType extends InputType {
     BaseModel?: InputModelType;
     DiscriminatorPropertyName?: string;
     DiscriminatorValue?: string;
+    IsInternal: boolean;
```

Comment:
Use `Accessibility` which is to define if the operation is public/internal/private. No need to add a new property.